ui/app/AppLayouts/Profile/helpers/ProfileShowcaseModelAdapter.qml

```diff
@@ -13,50 +13,78 @@ QObject {
 
     // Communities input models
     property alias communitiesSourceModel: communitySFPM.sourceModel
+    property alias communitiesShowcaseModel: communityJoinedModel.leftModel
 
     // adapted models
-    readonly property alias adaptedCommunitiesSourceModel: communitySFPM
+    readonly property alias adaptedCommunitiesSourceModel: communityJoinedModel
 
     // Accounts input models
     property alias accountsSourceModel: accountsSFPM.sourceModel
 
     // adapted models
     readonly property alias adaptedAccountsSourceModel: accountsSFPM
 
+    //helpers
+    property var isAddressSaved: (address) => false
```

Comment:
It's for the saved address flag in the delegate. We're adding here a new model role for this so that the delegate user won't have to import the wallet store.